client/src/components/Ratings.jsx

```diff
@@ -1,75 +1,48 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
-
 import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'underscore';
 import ReviewsList from './ReviewsList';
-// Don't have to import the server/index.js?
-
-
-
-// bring all reviews here to have data to go through
-
-// functions needed
-  // componentDidMount? Or alternative?
-
-  // calculate review to the nearest half star
-    // IMPORT Andrew's star function
-
-
-// calculate percentage of reviews recommend product
-// function getPercentageRecommended(results) {
-  // create new array of only 'true' values from results
-  // const trues = results.filter(
-  //   (resultItem.recommend) => (resultItem.recommend === true)
-  //     );
-
-  // return the Length of array of trues divided by original array length
-  // return Math.ceil(trues.length / ratingsArray.length) * 100;
-// }
 
+function getPercentageRecommended(results) {
+  const copy = results.map((result) => result.recommend);
+  const trues = copy.filter(
+    (resultItem) => (resultItem === true)
+  );
+  return Math.ceil(trues.length / results.length) * 100;
+}
 
-  // get average review value
-function getAverageReview(results) {
-  const reviewTotal = results.reduce(
+function averageToNearestTenth(array) {
+  const reviewVals = array.map(
+    (value) => value.rating
+  );
+  const reviewTotal = _.reduce(
+    reviewVals,
     (previousValue, currentValue) => previousValue + currentValue,
-    0
+    0,
   );
-
-  const { length } = results;
-  return (reviewTotal / length);
+  const number = (reviewTotal / array.length);
+  const whole = Math.floor(number);
+  const decimal = Math.trunc(((number % 1) * 10)) / 10;
+  return whole + decimal;
 }
 
-  // sort option event handler
-
-  // click handler for "More Reviews"
-
-// data needed
-  // results array from product review obj
-    // results length
-    // rating numbers
-    // recommended
-    // username
-    // date
-
-// props to pass down?
-  // individual review for ReviewsList
-
-
-function Ratings({ page, count, results }) {
-
+function Ratings({ results, setReviews }) {
   return (
     <div id="ratings-reviews-container">
 
-      {/* Big Rating and star count */}
-      {/* Use generateStars function from Overview */}
-      <h1 className="big-num"> 3.5 HardCode </h1>
+      <div>Ratings And Reviews</div>
 
-      {/* Individual product score and data */}
-      <div>
+      <h1 className="big-review-num">
```

Comment:
instead of giving this header a class you will use only once, use the selector:
```css
#ratings-reviews h1 {
}
```
to style this heading